# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-04-15 14:55+0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: authy/managers.py:24 authy/managers.py:26
msgid "The password must be set"
msgstr "The password must be set"

#: authy/models.py:17
msgid "active"
msgstr "active"

#: authy/models.py:20
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."

#: authy/serializers.py:74 authy/serializers.py:183 authy/services.py:123
#: contact_verification/serializers.py:26
msgid "Email or phone should be provided"
msgstr "Email or phone should be provided"

#: authy/serializers.py:85 authy/services.py:131
#: contact_verification/serializers.py:37
msgid "Enter email or phone not both"
msgstr "Enter email or phone not both"

#: authy/serializers.py:101
msgid "User with this email doesn't exist"
msgstr "User with this email doesn't exist"

#: authy/serializers.py:103
msgid "User with this phone doesn't exist"
msgstr "User with this phone doesn't exist"

#: authy/serializers.py:116
msgid "Incorrect password"
msgstr "Incorrect password"

#: authy/serializers.py:147
msgid "Invalid login credentials"
msgstr "Invalid login credentials"

#: authy/serializers.py:191
msgid "Passwords doesn't match"
msgstr "Passwords doesn't match"

#: authy/services.py:85
msgid "email verification message"
msgstr "Email verification code for MyApp: {code}"

#: authy/services.py:86
msgid "Email Verification Code"
msgstr "Email Verification Code"

#: authy/services.py:96
msgid "phone verification message"
msgstr "Phone verification code for MyApp: {code}"

#: authy/views.py:48
msgid "Logged in successfully."
msgstr "Logged in successfully."

#: authy/views.py:82
msgid "Registered successfully."
msgstr "Registered successfully."

#: common/custom_exceptions/custom_exception.py:15
#: common/custom_exceptions/custom_exception.py:17
#: common/response/view_response.py:148
msgid "A server error occurred."
msgstr "A server error occurred."

#: common/exception_handling/exception_handler_mapper.py:43
#: common/exception_handling/exception_handler_mapper.py:79
#: common/exception_handling/exception_handler_mapper.py:98
#: common/exception_handling/exception_handler_mapper.py:104
#: common/exception_handling/exception_handler_mapper.py:110
#: common/exception_handling/exception_handler_mapper.py:116
#: common/exception_handling/exception_handler_mapper.py:137
#: common/exception_handling/exception_handler_mapper.py:155
#: common/exception_handling/exception_handler_mapper.py:167
#: common/exception_handling/exception_handler_mapper.py:173
#: common/exception_handling/exception_handler_mapper.py:197
#: common/response/view_response.py:107
msgid "An error occured!"
msgstr "An error occured!"

#: common/exception_handling/exception_handler_mapper.py:65
msgid "Already exist!"
msgstr "Item already exists!"

#: common/exception_handling/exception_handler_mapper.py:86
#: common/exception_handling/exception_handler_mapper.py:161
#: common/exception_handling/exception_handler_mapper.py:185
#: user/services.py:26
msgid "Content not found!"
msgstr "Content not found!"

#: common/exception_handling/exception_handler_mapper.py:92
#: common/exception_handling/exception_handler_mapper.py:182
#: common/exception_handling/exception_handler_mapper.py:188
#: common/exception_handling/exception_handler_mapper.py:191
#: user/serializers.py:78
msgid "You don't have permission for this action!"
msgstr "You don't have permission for this action!"

#: common/exception_handling/exception_handler_mapper.py:122
#: common/exception_handling/exception_handler_mapper.py:170
#: common/response/view_response.py:130
msgid "Request parameters are not valid!"
msgstr "Request parameters are not valid!"

#: common/exception_handling/exception_handler_mapper.py:128
msgid "Bad request!"
msgstr "Bad request!"

#: common/exception_handling/exception_handler_mapper.py:131
msgid "Request aborted!"
msgstr "Request aborted!"

#: common/exception_handling/exception_handler_mapper.py:143
msgid "Upload is aborted!"
msgstr "Upload is aborted!"

#: common/exception_handling/exception_handler_mapper.py:149
msgid "Session interrupted!"
msgstr "Session interrupted!"

#: common/exception_handling/exception_handler_mapper.py:176
msgid "Authentication failed!"
msgstr "Authentication failed!"

#: common/exception_handling/exception_handler_mapper.py:179
#: common/exception_handling/exception_handler_mapper.py:200
msgid "Not authenticated!"
msgstr "Not authenticated!"

#: common/exception_handling/exception_handler_mapper.py:194
msgid "Unsupported media type!"
msgstr "Unsupported media type!"

#: common/response/view_response.py:76 contact_verification/views.py:77
#: contact_verification/views.py:108
msgid "Success"
msgstr "Success"

#: common/serializers.py:33 common/serializers.py:107
msgid ""
"The locale that refers to English language doesn't exist. Please create a "
"locale with 'en' code first!"
msgstr ""
"The locale that refers to English language doesn't exist. Please create a "
"locale with 'en' code first!"

#: common/serializers.py:42 common/serializers.py:116
msgid "Root object locale id must be {} which refers to English language"
msgstr "Root object locale id must be {} which refers to English language"

#: common/serializers.py:79 common/serializers.py:155
msgid ""
"If this is the first time to create a translation it should be in English"
msgstr ""
"If this is the first time to create a translation it should be in English"

#: common/services.py:28 common/services.py:99
msgid "Root must be provided"
msgstr "Root must be provided"

#: common/views.py:137 common/views.py:206 common/views.py:295
#: common/views.py:361 payment/views.py:182 user/views.py:185 user/views.py:288
#: user/views.py:406
msgid "Created successfully!"
msgstr "Created successfully!"

#: common/views.py:157 common/views.py:315 payment/views.py:202
#: user/views.py:132 user/views.py:308 user/views.py:426
msgid "Updated successfully!"
msgstr "Updated successfully!"

#: common/views.py:174 common/views.py:178 common/views.py:332
#: common/views.py:336 payment/views.py:219 payment/views.py:223
#: user/views.py:151 user/views.py:155 user/views.py:325 user/views.py:329
#: user/views.py:443 user/views.py:447
msgid "Deleted successfully!"
msgstr "Deleted successfully!"

#: contact_verification/services.py:45
msgid "Code is sent. Check your email."
msgstr "Code is sent. Check your email."

#: contact_verification/services.py:47
msgid "Code is sent. Check your messages."
msgstr "Code is sent. Check your messages."

#: contact_verification/services.py:66
msgid "Code is invalid or expired"
msgstr "Code is invalid or expired"

#: user/models.py:57
msgid "Personal"
msgstr "Personal"

#: user/models.py:58
msgid "Business"
msgstr "Business"

#: user/models.py:59
msgid "Internal"
msgstr "Internal"

#: user/models.py:218
msgid "Public"
msgstr "Public"

#: user/models.py:219
msgid "Private"
msgstr "Private"
